Topics/Tech_Stacks/Swift_Mapkit.md

```diff
@@ -0,0 +1,263 @@
+# Introduction to MapKit for SwiftUI
+
+SwiftUI is a very vast collection of features designed by Apple to make iOS development…swift-er. One of those handy features is it’s ability to make maps, specifically with MapKit. 
+
+**Requirements:** To use these MapKit features, you need to use Xcode 15 and later.
+
+## Table of contents
+### [What is MapKit?](#what-is-mapkit-1)
+### [Creating a map](#creating-a-map-1)
+### [Markers](#markers-1)
+### [Annotations](#annotations-1)
+### [Camera Position](#camera-position-1)
+### [Additional Resources](#additional-resources-1)
+
+## What is MapKit?
+MapKit is one of Apple’s API framework that allows iOS developers to build map-centered views and apps easily and efficiently. 
+
+Based on the [Apple documentation](https://developer.apple.com/documentation/mapkit/), MapKit can:
+
+- Embed maps directly into your app’s windows and views.
+- Add annotations and overlays to a map to call out points of interest.
+- Add LookAround capabilities to enable users to explore locations at street level.
+- Respond to user interactions with well known points of interest, geographical features, and boundaries.
+- Provide text completion to make it easy for users to search for a destination or point of interest.
```

Comment:
text completion in what context? i'm assuming in a search bar if you're building a navigation app?

Reply:
This is straight from the apple documentation! Maybe I can add some examples as to when you would use these

Reply:
That makes sense! I don't think it was super clear that this was pulled from their docs, so maybe add a short line?

Reply:
But yes you are correct.

Reply:
I have a line before hand that says based off the apple documentation with link then list the features. Do you think that I should add more?

Reply:
Apparently I can't read! You are absolutely correct and it's quite clear in its current state. This looks good to go!